api/controllers/case.js

```diff
@@ -311,7 +312,7 @@ async function getCaseById(req, res) {
     the_case.bookmarked = bookmarked.case;
     res.status(200).json({ OK: true, data: the_case });
   } catch (error) {
-    log.error("Exception in GET /case/%s => %s", caseId, error);
+    log.error("Exception in GET /case/undefined => %s", error);
```

Comment:
undefined?  Seems like the `catch` handler could be invoked even if `caseId` is defined.